test/functional/p2p_orphan_handling.py

```diff
@@ -41,6 +42,8 @@
 # for one peer and y seconds for another, use specific values instead.
 TXREQUEST_TIME_SKIP = NONPREF_PEER_TX_DELAY + TXID_RELAY_DELAY + OVERLOADED_PEER_TX_DELAY + 1
 
+DEFAULT_MAX_ORPHAN_TRANSACTIONS = 100
```

Comment:
Was this https://github.com/bitcoin/bitcoin/pull/31040#discussion_r1815889280 done in 5c299ecafe6f336cffa145d28036b04b87e26712? I could not find it.